packages/viewer/README.template.md

```diff
@@ -133,9 +133,7 @@ applyPolyfills().then(() => defineCustomElements(window)).then(() => main());
 
 ## Examples
 
-Check out our [examples] repository for more in-depth examples of how to use
-Vertex's Viewer SDK.
+Check out our [examples] directory for more in-depth examples of how to use Vertex's Viewer SDK.
```

Comment:
does that automatically link to the directory in github?

Reply:
Oh, no. I can do that.

Reply:
Updated